### ISSUE TITLE
[ERROR] Error updating database.. Run with -vvv to see specific error info.

### ISSUE THREAD

**Issue by xyychenchen** [shlinkio/shlink#2320]
### Shlink version

4.4.0

### PHP version

8.3

### How do you serve Shlink

Self-hosted Apache

### Database engine

MariaDB

### Database version

5.5.68

### Current behavior

i run the installtion script and it keep saying [ERROR] Error updating database.. Run with -vvv to see specific error info. 
i am pretty sure that the info about the mariadb database is correct, i try to set using unix socket and the port, either way won't work, 
and i also try to run with -vvv, but i did not see anything info helpful
please tell me how to find the error, i really have no idea where went wrong, 
after run the set option to re-type the database info, i run vendor/bin/shlink-installer update,and it always shows

[root@ecs-TZWsG shlink]# vendor/bin/shlink-installer update
 Welcome to Shlink!!
 This tool will guide you through the installation process.

 Do you want to import configuration from previous installation? (You will still be asked for any new config option that did not exist in previous shlink versions) (yes/no) [yes]:
 > 

 Previous shlink installation path from which to import config:
 > /data/shlink/

 Custom configuration properly generated!

Updating database...

                                                                                                                        
 [ERROR] Error updating database.. Run with -vvv to see specific error info.                                            
                                                                                                                        

Generating proxies... Success!
Clearing entities cache... Success!
Downloading GeoLite2 db file... Success!
Downloading RoadRunner binary... Success!

and by the way, i have been successfully generate an API key yet, my first installtion failed with the database and geoIP key , after I  re-input the geoIP key , what is left is the database, keep saying error , i spent a day to fix this , not working still 

### Expected behavior

run the installion script supposed to run successfully

**Comment by acelaya** [comment]
Please, provide the output of the command when run with `-vvv`

**Comment by xyychenchen** [comment]
> Please, provide the output of the command when run with `-vvv`

output is the same as running `vendor/bin/shlink-installer install` , so i do not know how it can help identify the issues as the prompt indicate

**Comment by xyychenchen** [comment]
> Please, provide the output of the command when run with `-vvv`

```[root@ecs-TZWsG shlink]# vendor/bin/shlink-installer install -vvv
 Welcome to Shlink!!
 This tool will guide you through the installation process.

SERVER
======

 Select the runtime you are planning to use to serve Shlink (this is only used to conditionally skip some follow-up questions) [RoadRunner]:
  [0] RoadRunner
  [1] Classic web server (Nginx, Apache, etc)
 > 
```

like this

**Comment by acelaya** [comment]
I'm going to need the full output, not just the first question

**Comment by xyychenchen** [comment]
> I'm going to need the full output, not just the first question

you mean the full output when I finish the all questions of the installtion script?

**Comment by acelaya** [comment]
Yes. The installer runs a couple subprocesses at the end (like the one which is failing), and will print extra info when run in verbose mode.

**Comment by acelaya** [comment]
Are you trying to create a new Shlink installation, or updating an existing one?

**Comment by acelaya** [comment]
The root of the issue is this when running database migraine: `Unknown table 'CHECK_CONSTRAINTS' in information_schema`, which is likely caused because you are using a veeeery old MariaDB version, and I don't think the database abstraction library supports it. 

I'd suggest making sure you use a more recent version. The last one is 11.7, just as a reference.

**Comment by xyychenchen** [comment]
> Are you trying to create a new Shlink installation, or updating an existing one?

i ran installtion for several times before, and ran update several times before, because it kept failed
